cirq/protocols/has_stabilizer_effect_protocol.py

```diff
@@ -17,17 +17,19 @@
     Optional,
 )
 
+from cirq.ops.clifford_gate import SingleQubitCliffordGate
+
+from cirq import protocols
+
 
 def has_stabilizer_effect(val: Any) -> bool:
     """
-    Returns whether the input has a stabilizer effect. Currently only limits to
-    Pauli, H, S, CNOT and CZ gates and their Operations. Does not attempt to
-    decompose a gate into supported gates. For e.g. iSWAP or X**0.5 gate will
-    return False.
+    Returns whether the input has a stabilizer effect.
```

Comment:
I'd keep the caveats still (but update them) because even after this, it doesn't generalize to include all stabilizer-effect inputs.

Reply:
Rephrazed. I don't think we need to list all gates for which it returns True, because then we will have to update this pydoc every time _has_stabilizer_effect_ is defined for a new gate.